### PR TITLE
fix: husky.config.js not .huskyrc.config.js

### PR DESCRIPTION
**Pull request by christophehurpeau**
in [readme](https://github.com/typicode/husky/blob/master/README.md#upgrading-from-014) it says:

> Starting with 1.0.0, husky can be configured using .huskyrc, .huskyrc.json, .huskyrc.js or husky.config.js file.

### CODE REVIEW

**Comment by christophehurpeau** [comment]
bug introduced with https://github.com/typicode/husky/commit/015f3aae2ff14b5e0a71e2dd73adebdc0bf07749#diff-9fde8338c5cc7fec9e2341da618aa311 in  v4.2.2

**Comment by typicode** [comment]
Thanks!

File: sh/husky.sh [hunk 1]
```diff
@@ -58,7 +58,7 @@ debug "Current working directory is $(pwd)"
 # Don't skip if .huskyrc.js or .huskyrc.config.js are used as the heuristic could
```
Comment (vsn4ik) [inline sh/husky.sh]: `.huskyrc.config.js` -> `husky.config.js`

File: src/installer/__tests__/__snapshots__/scripts.ts.snap [hunk 2]
```diff
@@ -87,7 +87,7 @@ debug \\"Current working directory is $(pwd)\\"
 # Don't skip if .huskyrc.js or .huskyrc.config.js are used as the heuristic could
```
Comment (vsn4ik) [inline src/installer/__tests__/__snapshots__/scripts.ts.snap]: `.huskyrc.config.js` -> `husky.config.js`

**Pull request merged**